app/models/import_trade_summary.rb

```diff
@@ -0,0 +1,12 @@
+require 'api_entity'
+
+class ImportTradeSummary
+  include ApiEntity
+
+  collection_path '/import_trade_summary'
```

Comment:
It's actually misdirecting to have this/you don't need it. This is used to construct the path to pull out the resource (e.g `/geographical_areas`)

Reply:
👍 I agree, I added "just in case", but we don't need it.